jsk_aibo_robot/aibo_selenium_ros/python/aibo_selenium_ros/__init__.py

```diff
@@ -0,0 +1,179 @@
+import base64
+import io
+import logging
+import random
+import time
+
+import selenium
+import undetected_chromedriver as uc
+from imageio import imread
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+
+logger = logging.getLogger(__name__)
+
+
+class AIBOBrowserInterface(object):
+
+  def __init__(self,
+               webdriver_path='/usr/bin/chromedriver',
+               chrome_executable_path='/usr/bin/chromium-browser',
+               login_page_url='https://myaibo.aibo.sony.jp/#/',
+               login_id='',
+               login_pw='',
+               auto_login=True,
+               headless=False,
+               timeout=20.):
+
+    self.initialized = False
+
+    options = Options()
+    if headless:
+      options.add_argument('--headless')
+    if chrome_executable_path is not None:
+      options.binary_location = chrome_executable_path
+
+    self.driver = uc.Chrome(executable_path=webdriver_path, options=options, version_main=115)
```

Comment:
chrominum version is different for each system. So it will be convenient if this program retrieve main version from `chromium-browser --version` command. See https://github.com/jsk-ros-pkg/jsk_robot/pull/1798/commits/7cbcd36e9480f871469fd1adafe0a1e39e914218 for reference.